.editorconfig

```diff
@@ -0,0 +1,27 @@
+; This file is for unifying the coding style for different editors and IDEs.
+; More information at http://editorconfig.org
+root = true
+
+[*]
+indent_style = space
+end_of_line = lf
+insert_final_newline = true
+trim_trailing_whitespace = true
+charset = utf-8
```

Comment:
```suggestion
charset = utf-8
indent_size = 4
```

Adding `indent_size` here removes the need for configuring it for different extensions.